distributions/nuxt-ts/bin/nuxt-ts.js

```diff
@@ -5,10 +5,11 @@ process.env.NUXT_TS = 'true'
 
 const nuxtCommands = ['dev', 'build', 'generate', 'start']
 const rootDir = (process.argv[2] && !nuxtCommands.includes(process.argv[2])) ? process.argv[2] : process.cwd()
+const tsConfigPath = require('path').resolve(rootDir, 'tsconfig.json')
```

Comment:
Prefer moving require/import to top of the script, when lazy loading is not happening

Reply:
I don't think `import { resolve } from 'path'`  will work as it's the binary and not compiled.

So just `const { resolve } = require('path')` at the top should be ok ?

Reply:
@kevinmarrec YES :)